code/game/area/areas.dm

```diff
@@ -451,6 +453,21 @@
 		if(newarea==oldarea) return
 
 		M.lastarea = src
+		if(istype(M) && M.client)
+			var/loading_times = pick(2 SECONDS, 3 SECONDS, 3.5 SECONDS)
+			if(prob(1))
+				loading_times = 2 MINUTES
```

Comment:
```suggestion
				loading_times = 30 SECONDS
```
God please sods law will kick in

Reply:
Yeah we should make it take 30 minutes I agree